.stylelintrc

```diff
@@ -13,7 +13,6 @@
       "darken"
     ],
     "property-disallowed-list": [
-      "border-radius",
```

Comment:
You don't have to, simply use `@include border-radius($value, $value)` to pass  through contraints.

Reply:
The problem is that in this mixin, there is $enable-rounded which is disabled for the project...

Reply:
There's a condition to pass through it if a second argument is set.